tests/modules/core/lib/Auth/Process/CardinalitySingleTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+// Alias the PHPUnit 6.0 ancestor if available, else fall back to legacy ancestor
+if (class_exists('\PHPUnit\Framework\TestCase', true) and !class_exists('\PHPUnit_Framework_TestCase', true)) {
```

Comment:
The only thing I'm not sure about is the necessity of this if-statement...
Why not use the namespaced version after `extends`? Is this some backwards compatibility-thing for 1.14?